### PR TITLE
Add isogram exercise

### PR DESCRIPTION
**Pull request by pfertyk**

### CODE REVIEW

**Comment by pfertyk** [comment]
@BNAndras I basically copied the tests from Python :stuck_out_tongue: Currently, the example handles only spaces and dashes, because only they are mentioned in the description of this exercise. Should I look for a more sophisticated solution, or can we leave it as is?

**Comment by BNAndras** [comment]
An example only needs to prove the exercise can be solved using the current test suite. It's not public-facing at all for a practice exercise. For a concept exercise, the exemplar (the equivalent of the example) should be idiomatic since it's shown to mentors as a possible reference solution to guide students towards. On the other hand, sometimes students do check the example file after they complete the exercise. I wouldn't sweat it too much. Future changes might bring different tests, and at that point, you'll know for sure what needs to be changed in the example.

File: exercises/practice/isogram/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "authors": [],
```
Comment (BNAndras) [inline exercises/practice/isogram/.meta/config.json:2]: I'd add your GitHub username here.

**Review by BNAndras — CHANGES_REQUESTED** [review]

File: exercises/practice/isogram/.meta/config.json (hunk 1 above)
Comment (pfertyk) [reply exercises/practice/isogram/.meta/config.json]: Good point, thanks! :sweat_smile:

**Review by BNAndras — APPROVED** [review]

**Pull request merged**